pennylane/circuit_graph.py

```diff
@@ -647,7 +671,17 @@ def has_path(self, a, b):
         Returns:
             bool: returns ``True`` if a path exists
         """
-        return nx.has_path(self._graph, a, b)
+        if a == b:
+            return True
+
+        return (
+            len(
+                rx._digraph_dijkstra_shortest_path(
```

Comment:
I missed this the first time around, but how come this is a private function? Is it not part of the RX public API?

Reply:
This is not part of the [public API](https://qiskit.org/documentation/retworkx/api.html) right now. I found it by looking into the code, but I am expecting to be added as a new feature within next releases.